OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Validations.Modify.cs

```diff
@@ -96,7 +96,8 @@ public async Task ShouldThorwValidationExceptionOnModifyWhenTeacherIsNullAndLogI
 
             invalidTeacherException.AddData(
                 key: nameof(Teacher.UpdatedDate),
-                values: "Date is required");
+                "Date is required",
```

Comment:
Parameter aliasing

Reply:
@ShriHumrudha  it's not possible here because we have 2 strings.